src/ephy-bookmarks-popover.c

```diff
@@ -0,0 +1,530 @@
+/* -*- Mode: C; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/*
+ * Copyright © 2016 Iulian-Gabriel Radu <iulian.radu67@gmail.com>
```

Comment:
If you want an @gnome.org mail alias, they're pretty easy to set up with GMail. You're a Foundation member, right? That's the only requirement.